live_validation_scripts/validate_live_pixel.mjs

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+
+import csv from 'csv-parser';
+import fs from 'fs';
+
+import { getArgParserWithCsv } from '../src/args_utils.mjs';
+import { ParamsValidator } from '../src/params_validator.mjs';
+import { LivePixelsValidator } from '../src/live_pixel_validator.mjs';
+
+import * as fileUtils from '../src/file_utils.mjs';
+
+const argv = getArgParserWithCsv('Validates pixels from the provided CSV file', 'path to CSV file containing pixels to validate').parse();
+
+function main(mainDir, csvFile) {
+    console.log(`Validating live pixels in ${csvFile} against definitions from ${mainDir}`);
+
+    const productDef = fileUtils.getProductDef(mainDir);
+    const forceLowerCase = productDef.forceLowerCase;
+
+    const commonParams = fileUtils.getCommonParams(mainDir, forceLowerCase);
+    const commonSuffixes = fileUtils.getCommonSuffixes(mainDir, forceLowerCase);
+
+    const tokenizedPixels = fileUtils.getTokenizedPixels(mainDir, forceLowerCase);
+    const paramsValidator = new ParamsValidator(commonParams, commonSuffixes);
+    const ignoreParams = fileUtils.getIgnoreParams(mainDir, forceLowerCase);
```

Comment:
These 'getters' are reading files, so I'd suggest reflecting that in the naming, e.g. `loadCommonParams`, or `readCommonParams`.